NAMESPACE

```diff
@@ -18,15 +23,12 @@ S3method(score,forecast_binary)
 S3method(score,forecast_point)
 S3method(score,forecast_quantile)
 S3method(score,forecast_sample)
-S3method(validate_forecast,default)
-S3method(validate_forecast,forecast_binary)
```

Comment:
does validate_forecast need to be a s3 method now or can it just depend on assert_forecast dispatching underneath?